example-playbooks/replace_node/replace_node.yml

```diff
@@ -164,16 +181,6 @@
 
     - name: Set replace_node_first_boot
       block:
-        - name: Get the host id for all nodes
-          uri:
-            url: "http://{{ scylla_api_address }}:{{ scylla_api_port }}/storage_service/host_id"
-            method: GET
-          register: _host_ids
-          until: _host_ids.status == 200
-          retries: 5
-          delay: 1
-          delegate_to: "{{ temporary_seed }}"
-
```

Comment:
Removing this block will break the `set_fact` below because you are iterating over a variable from a different play (`_host_ids`).

And according to the documentation "registered" variables scope is a corresponding play: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#registering-variables

Reply:
The documentation actually says that they are valid for the rest of the playbook, including subsequent plays within the same playbook run:
> Registered variables are stored in memory. You cannot cache registered variables for use in future playbook runs. Registered variables are only valid on the host for the rest of the current playbook run, including subsequent plays within the same playbook run.

Reply:
Anyway, this task is not being moved to the other play anymore